src/objective-c/tests/GRPCClientTests.m

```diff
@@ -303,7 +303,7 @@ - (void)testExceptions {
 
   // Try to set parameters to nil for GRPCCall. This should cause an exception
   @try {
-    GRPCCall *call = [[GRPCCall alloc] initWithHost:nil
+    GRPCCall *call __unused = [[GRPCCall alloc] initWithHost:nil
```

Comment:
Removed object `call` and cast the expression to void to eliminate the 'expression result unused' warning.